jwcrypto/jws.py

```diff
@@ -30,7 +30,8 @@
     'RS256', 'RS384', 'RS512',
     'ES256', 'ES384', 'ES512',
     'PS256', 'PS384', 'PS512',
-    'EdDSA', 'ES256K']
+    'EdDSA', 'ES256K',
+    'BP256R1', 'BP384R1', 'BP512R1']
```

Comment:
Until these are IANA registered names I do not think I want them as default algorithms.

Reply:
Understandable. Is there a way to add Algorithms at runtime? I haven't found one using the public API...

Reply:
https://jwcrypto.readthedocs.io/en/latest/jws.html#jwcrypto.jws.JWS.allowed_algs

Also see the signature od the add_signature() function or the deserialize function, they both support an alg parameter.

You can look in tests.py to see how this is used.